.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!--
+⚠️ If you do not respect this template, your pull request will be closed.
+⚠️ Your pull request title should be short detailed and understandable for all.
+⚠️ Also, please add a release note file using reno if the change needs to be
+  documented in the release notes.
+⚠️ If your pull request fixes an open issue, please link to the issue.
+
+- [ ] I have added the tests to cover my changes.
```

Comment:
This repo doesn't have tests yet, so I think we should remove this line and add it again once we have the test infrastructure ready for adding more test, what do you think?